plugins/language_nix.lua

```diff
@@ -0,0 +1,87 @@
+-- mod-version:3
+-- https://nixos.wiki/wiki/Overview_of_the_Nix_Language
+local syntax = require "core.syntax"
+
+local function merge_tables(a, b)
+  for _, v in pairs(b) do
+    table.insert(a, v)
+  end
+end
+
+local default_symbols = {
+  ["import"]   = "keyword2",
+  ["with"]     = "keyword2",
+  ["builtins"] = "keyword2",
+  ["inherit"]  = "keyword2",
+  ["assert"]   = "keyword2",
+  ["let"]      = "keyword2",
+  ["in"]       = "keyword2",
+  ["rec"]      = "keyword2",
+  ["if"]       = "keyword",
+  ["else"]     = "keyword",
+  ["then"]     = "keyword",
+  ["true"]     = "literal",
+  ["false"]    = "literal",
+  ["null"]     = "literal",
+}
+
+local default_patterns = {}
+
+local string_interpolation = {
+  { pattern = {"%${", "}"}, type = "keyword2", syntax = {
+    patterns = default_patterns,
+    symbols = default_symbols,
+  }},
+  { pattern = "[%S][%w]*", type = "string" },
+}
+
+merge_tables(default_patterns, {
+  { pattern = "#.-\n",        type = "comment" },
```

Comment:
I just noticed that many other language plugins use `.-\n`.
This is not needed, `.*` should be enough.